scripts/build_utils.sh

```diff
@@ -1351,6 +1337,16 @@ setup_rpm_build_deps() {
         $SUDO dnf config-manager --add-repo http://apt-mirror.front.sepia.ceph.com/lab-extras/8/
         $SUDO dnf config-manager --setopt=apt-mirror.front.sepia.ceph.com_lab-extras_8_.gpgcheck=0 --save
 
+    elif [ "$RELEASE" = 9 ]; then
+        $SUDO dnf -y copr enable ceph/el9
+        $SUDO dnf config-manager --setopt=copr\:copr.fedorainfracloud.org\:ceph\:el9.gpgcheck=0 --save
+
+        $SUDO dnf -y install epel-next-release
+
+        $SUDO dnf config-manager --add-repo http://mirror.stream.centos.org/9-stream/CRB/x86_64/os/
+        $SUDO dnf config-manager --setopt=mirror.stream.centos.org_9-stream_CRB_x86_64_os_.gpgcheck=0 --save
```

Comment:
Why do we need to add these repos? I'm also wondering why we disable GPG checking.

Reply:
We need CRB for librdkafka-devel and the el9 copr repo for the thrift-devel 

There was a time when enabling gpgcheck for these two repos caused package installation failures, but that doesn't appear to be the case any more

Reply:
Oh, I see now. In that case, you can enable the CRB definition that is already in `/etc/yum.repos.d`. This command works in the `centos:stream9` container:
```
dnf config-manager --set-enabled crb
```